Automattic/jetpack
# remove Jetpack dependencies from the Connection package

This PR removes some Jetpack-specific XMLRPC endpoints from the Connection package and places it in the Jetpack plugin.

This should not change anything for sites with the Jetpack plugin active.

#### Changes proposed in this Pull Request:

* Removes Jetpack specific XMLRPC endpoints from the Connection package

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


1164484600834010-as-1199928370054538

#### Testing instructions:






* Checkout this branch
* Connect the site
* Run the Jetpack Debugger and make sure Everything looks great
* In the debugger, scroll down to "Tests"
* Click on "success: test authenticated xmlrpc request for jetpack passes connection test"
* Make sure the response is the Jetpack version, and not the `success` string
* Test requests to the json_api endpoint
* Go to Jetpack Dashboard and Disconnect. Make sure disconnection still works


#### Proposed changelog entry for your changes:


* Jetpack plugin extends the Connection package and registers its own XMLRPC methods
